.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -7,35 +7,15 @@ on:
     branches:
```

Comment:
@manh-t 

> This workflow will synchronize the .github/wiki folder with your project's Github Wiki.

If I understand correctly, we need to:
1.  Update `docs/**` with `.github/wiki/**` in `publish_docs_to_wiki.yml` 
2. Move everything inside `docs` to `.github/wiki` 
3. Remove `docs` folder

Reference: https://github.com/nimblehq/ios-templates/pull/299/files

What do you think? 🙏

Reply:
@Tuubz I'm aware of this, and I'm going to create an issue to move `docs` to `.github/wiki/**` instead of refactoring them in this PR. 🙆‍♂️

Reply:
Oh, any reason we don't do it in this PR? Seems like an easy fix

If we merge this now, then we would be merging broken code 💭

Reply:
> If we merge this now, then we would be merging broken code

@Tuubz ah true that I forgot this. Okay I'll update it 😁

Reply:
@Tuubz updated 🙏